src/components/footer/LanguageNavigation.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+import { Link, StackDivider } from '@chakra-ui/react';
```

Comment:
Does it make sense to extract the `StackDivider` component?

Reply:
Don't we have already the `Link` component?

Reply:
Regarding the extraction, is there something that changed in the strategy for extracting chakra components?
I had in mind that we are fine using them internally and if we wanna export them to make them usable by the consumer we create a custom theme for them. If we have a custom-themed one we shall use the custom-themed component over the chakra one.
Why would we do that now within the footer task and not for the other components we use the container?